src/implementations/twig/components/link/link.html.twig

```diff
@@ -76,7 +78,17 @@
     size: '2xs',
   })] %}
 
-  {% set _link = _link|merge({ icon_position: _link.icon_position|default('after') }) %}
+  {% if _link.sr_external is not empty or extra_accessibility.title is not empty %}
```

Comment:
mmh..it seems more logical to me to do this:
```
  {% if _extra_accessibility.title is empty and _link.sr_external is not empty %}
    {% set _extra_accessibility = _extra_accessibility|merge({ title: _link.sr_external }) %}
  {% endif %}
```
Basically we would use the new parameter only in case this is not passed the "old way". Currently with the code  we have we could still generate an empty title tag in the icon, with this code we wouldn't.
Furthermore we are merging the object so whatever is passed in it other than the title would be preserved.
If you follow this suggestion _extra_accessibility needs to be defined as usual with a merge using the default parameter and then there is no need to set again the default when passing it in the include.